kitsune/users/models.py

```diff
@@ -227,6 +227,10 @@ def answer_helpfulness(self):
     def is_subscriber(self):
         return self.products.exists()
 
+    @property
+    def is_staff(self):
+        return self.user.groups.filter(name="Staff").exists()
```

Comment:
Let's add the name in the settings. Eg SUMO_STAFF="Staff" and use `settings.SUMO_STAFF` here

Reply:
Ok. It seems weird to me to have SUMO_STAFF be a placeholder for Staff. But the idea of that group being more configurable makes sense. Done.